examples/server/app.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express')
+const path = require("path")
+const app = express()
+const port = 3000
+
+app.use(express.static(path.join(__dirname, 'public')));
+app.engine('ejs', require('ejs').__express);
+
+app.set('view engine', 'ejs');
+
+app.get('/', (req, res) => {
+    res.render('parquetFiles', {
+        // files: "fruits, fruits-no-index".split(','),
```

Comment:
This should be removed or commented with a reason why is should remain.

Reply:
good catch thanks